packages/protocol/test/governance/proposals.ts

```diff
@@ -0,0 +1,56 @@
+import { assertEqualBN } from '@celo/protocol/lib/test-utils'
+import { toFixed } from '@celo/utils/lib/fixidity'
+import BigNumber from 'bignumber.js'
+import { ProposalsTestContract, ProposalsTestInstance } from 'types'
+
+const ProposalsTest: ProposalsTestContract = artifacts.require('ProposalsTest')
+
+// @ts-ignore
+// TODO(mcortesi): Use BN
+ProposalsTest.numberFormat = 'BigNumber'
+
+contract('ProposalsTest', () => {
+  let proposalsTest: ProposalsTestInstance
+
+  describe('#getSupportWithQuorumPadding()', () => {
+    const networkWeight = 100
+    const quorum = new BigNumber(5).div(new BigNumber(10))
```

Comment:
Is this value ever used directly, or is it always `toFixed`'d first? If the latter, consider declaring this like other Fixidity values `= toFixed(5 / 10)`.